crypto/sr25519/bal.go

```diff
@@ -0,0 +1,76 @@
+package sr25519
```

Comment:
There shouldn't be any need to change this package. The `sr25519` package is just for working with the keys. In the case of balance that can not be determined from a key alone. Balance comes from the protocol client.